spec/appcenter_upload_spec.rb

```diff
@@ -1561,6 +1561,34 @@ def stub_add_to_destination(status, app_name = "app", owner_name = "owner", dest
       expect(values[:dsym_path]).to eq('./spec/fixtures/symbols/Themoji.dSYM.zip')
     end
 
+    it "rejects dsym for a non-Apple app file" do
+      expect do
+        stub_check_app(200)
+        stub_create_release_upload(200)
+        stub_upload_build(200)
+        stub_update_release_upload(200, 'committed')
+        stub_update_release(200)
+        stub_get_destination(200)
+        stub_add_to_destination(200)
+        stub_get_release(200)
+        stub_create_dsym_upload(200)
+        stub_upload_dsym(200)
```

Comment:
since we are expecting the error before dsym upload, let's not stub dsym related calls.